test/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter_test.cc

```diff
@@ -865,7 +865,7 @@ TEST_F(GrpcJsonTranscoderFilterTest, TranscodingUnaryError) {
 
   EXPECT_EQ(Http::FilterDataStatus::StopIterationNoBuffer, filter_.decodeData(request_data, true));
   EXPECT_EQ(0, request_data.length());
-  EXPECT_EQ(decoder_callbacks_.details(), "grpc_json_transcode_failure{INVALID_ARGUMENT}");
+  EXPECT_EQ(decoder_callbacks_.details(), "grpc_json_transcode_failure{INVALID_ARGUMENT:_}");
```

Comment:
What does the new ":_" suffix mean?

Reply:
This is introduced by `MessageUtil::codeEnumToString`. Maybe replace it with `absl::StatusCodeToString(code)`.

https://github.com/envoyproxy/envoy/blob/b4e61389f7db842a7bcee89d0fc080b23b27a9d2/source/common/protobuf/utility.cc#L352

Reply:
Your guess is as good as mine. This is details of protobuf implementation.